docker/build.mk

```diff
@@ -287,7 +287,7 @@ debug-$(1)-$(2): .$(1)-$(2)-oss-fuzz-runner
     -e BENCHMARK=$(2) \
     -e FUZZ_TARGET=$($(2)-fuzz-target) \
     --entrypoint "/bin/bash" \
-    -it $(BASE_TAG)/oss-fuzz/runners/$(1)/$(2)
+    -it $(BASE_TAG)/unners/$(1)/$(2)
```

Comment:
typo: runners ?

Can you try running the make run, make debug on 1-2 ossfuzz targets.

Reply:
`make run` works, this was a seperate comand that would have failed.